### DISCUSSION TITLE
Seems that all threads are expanded by default in Thunderbird 115. How can I collapse them?

### DISCUSSION THREAD

**Discussion by x-ji** [protz/thunderbird-conversations#2033] [Q&A]
Seems that all threads are expanded upon Thunderbird starting up for some reason. Toggling "By default, expand: No messages" in settings also didn't help. How can I collapse all threads (manually or automatically)?

**Comment by Standard8** [comment]
You'll want to raise this with the Thunderbird team at https://support.mozilla.org/products/thunderbird - the Conversations' add-on does not control the thread pane, except on initial installation where it sets what it can to threaded.

From then on, Thunderbird is responsible for the thread pane and the settings there.

`By default, expand: No messages` applies only to the message pane, where the conversations are viewed.
